ios/RNOneginiSdk.swift

```diff
@@ -357,28 +367,52 @@ class RNOneginiSdk: RCTEventEmitter, ConnectorToRNBridgeProtocol {
 
     @objc
     func enrollMobileAuthentication(_ resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) {
-        let completion = makeCompletionHandler(resolver: resolve, rejecter: reject)
-        bridgeConnector.toMobileAuthConnector.mobileAuthHandler.enrollForMobileAuth(completion)
+        bridgeConnector.toMobileAuthConnector.mobileAuthHandler.enrollForMobileAuth { result in
+            switch result {
+            case .success:
+                resolve(nil)
+            case .failure(let error):
+                self.rejectWithError(reject, error)
```

Comment:
is `self` here needed? 

here and other places

Reply:
yes it's needed